cmd/apiProxyUpload_test.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRunApiProxyUpload(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Successfull Api Proxy Create Test", func(t *testing.T) {
+		filesMock := mock.FilesMock{}
+		path := filepath.Join("tempDir", "apiproxy.zip")
+		filesMock.AddFile(path, []byte("dummy content"))
+		exists, err := filesMock.FileExists(path)
+		assert.NoError(t, err)
+		assert.True(t, exists)
```

Comment:
golang asserts don't stop the test run execution. So if the file does not exist it will still run. Maybe there is a way to stop the execution here if the assert is negative? Otherwise you need to check with an if and stop the execution manually.

Reply:
fixed with if condition

Reply:
Same applies for the check if the file exists or not. Asserts are rather used for tests, but you are not testing whether the file `filesMock.AddFile(...)` function works, or?
The same applies for the second test!